core/memory_tracker/cc/memory_tracker.h

```diff
@@ -16,16 +16,18 @@
 
 #include "core/cc/target.h"
 
-#if (TARGET_OS == GAPID_OS_LINUX) || (TARGET_OS == GAPID_OS_ANDROID)
 #define COHERENT_TRACKING_ENABLED
 
 #ifndef GAPII_MEMORY_TRACKER_H
 #define GAPII_MEMORY_TRACKER_H
 
+#if (TARGET_OS == GAPID_OS_LINUX) || (TARGET_OS == GAPID_OS_ANDROID)
```

Comment:
Consider splitting into separate files under `core/memory_tracker/cc/posix`, `core/memory_tracker/cc/windows` like we do in other directories.